test/test_constants.dart

```diff
@@ -1,7 +1,8 @@
 import 'package:openfoodfacts/openfoodfacts.dart';
 
 class TestConstants {
-  static const UserAgent TEST_USER_AGENT = UserAgent(
+  // ignore: non_constant_identifier_names
+  static UserAgent TEST_USER_AGENT = UserAgent(
```

Comment:
Just curious: in which cases would it make sense to change the value? `const` seemed appropriate, didn't it?

Reply:
As I've previously mentioned, the `trim()` forces us to remove the `const`

Reply:
@g123k `final` would probably work, then.

Reply:
Indeed, you're totally right